lib/src/Uncertainty/Algorithm/MetaModel/FunctionalChaos/FunctionalChaosSobolIndices.cxx

```diff
@@ -65,111 +66,119 @@ inline bool varianceComparison(const std::pair<UnsignedInteger, Scalar> &a, cons
   return a.second > b.second;
 }
 
+/* String formatter with blank space right padding */
+String FunctionalChaosSobolIndices::padString(const String & string) const
+{
+  const String intermediateString = OSS() << string;
```

Comment:
`const String intermediateString(OSS() << string);` I this different from `const String intermediateString(string);`? If not, why do you need a copy?

Reply:
Fixed at 17b243fba